MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +221,189 @@ this.Content = chart;
 
 {% endtabs %}
 
+### Trackball Label Template
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [ChartSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html).
```

Comment:
@EzhilarasanElangovan31 TrackballLabelTemplate exposed in Cartesian Series not Chart series. Modify it

Reply:
Modified to Cartesian Series